pyan/__init__.py

```diff
@@ -49,6 +50,7 @@ def create_callgraph(
         grouped_alt: if to use alternative grouping
         annotated: if to annotate graph with filenames
         grouped: if to group by modules
+        max_iter: maximum number of iterations for filtering. Defaults to 1000.
```

Comment:
Technically, this looks like something for a separate PR? (I don't mind, though - at this point, it's probably not worth the time cost to separate it.)

Reply:
Agree on both questions